src/CalculatorButtonValues.elm

```diff
@@ -13,13 +13,21 @@ type CalButton
     | CalNumNine
     | CalNumZero
     | CalDecimal
+    | CalAdd
+    | CalEqual
 
 
 type ButtonType
     = Num
     | Decimal
 
 
+type Action
+    = Blank
```

Comment:
`Blank` is an awfully strange `Action` to support. Are you perhaps wanting a `Maybe` somewhere?

Reply:
Did you want to address `Blank` in this PR or a separate one?